libs/martech/martech.js

```diff
@@ -190,8 +182,23 @@ const setupEntitlementCallback = () => {
   );
 };
 
+function isProxied() {
+  // return window.location.hostname.endsWith('adobe.com');
+  const { hostname } = window.location;
+  return (
```

Comment:
I think this would work better
```suggestion
  return /^(www|milo|business|blog)(\.stage)?\.adobe.com$/.test(hostname)
```

Reply:
nit:  `return /^(www|milo|business|blog)(\.stage)?\.adobe\.com$/.test(hostname)
`